index.md

```diff
@@ -19,13 +19,28 @@ Data Carpentry's aim is to teach researchers basic concepts, skills, and tools f
 {: .prereq}
 
 > ## Data
->
-> The data for this workshop are is the [FIXME](FIXME) available on FigShare, with a CC-BY license available for reuse.
->
-> FIXME add a short description about the dataset.
->
+> Data for this lesson comes from the National Ecological Observatory Network. These data files represent teaching version of the data, 
+> with sufficient complexity to teach many aspects of  data analysis and management, but with many complexities removed to allow students
+> to focus on the core ideas and skills being taught.  
+> 
+> There are four data sets included, all of which are available 
+> [on Figshare](https://figshare.com/articles/Spatio_temporal_Series_Teaching_Data_Subsets/2009586)
+> under a CC-BY license. You can download all of the data used in this workshop by clicking 
+> [this download link](https://ndownloader.figshare.com/articles/2009586/versions/10). 
+> Clicking the download link will automatically download all of the files to your default download directory as a single compressed
+> (`.zip`) file. To expand this file, double click the folder icon in your file navigator application (for Macs, this is the Finder 
+> application).
+> 
+> | Dataset | File name | Description |
+> | ---- | ------| ---- | 
+> | Site layout shapefiles | NEON-DS-Site-Layout-Files.zip | A set of shapefiles for the NEON's Harvard Forest field site and US and (some) state boundary layers. | 
+> | Meteorological data |  NEON-DS-Met-Time-Series.zip | Precipitation, temperature and other variables collected from a flux tower at the NEON Harvard Forest site
```

Comment:
@mjones01 can help here but the met data as i remember are not NEON data. They are from the LTER tower (i think??) gosh it's been a while. The language you have is correct @ErinBecker  but the source of the data i believe is harvard LTER which happens to be where there is a NEON site.

Reply:
@ErinBecker  and @lwasser   - this hackathon was in October 2015 and NEON had not yet released many of their data products.  As a result, Leah pulled the met data from the (Fisher Tower)[http://harvardforest.fas.harvard.edu:8080/exist/apps/datasets/showData.html?id=hf001] at Harvard Forest as data that was representative of what NEON would be producing. In the NEON version of the lessons, we link to that page in the metadata lesson, I haven't kept track enough of the (awesome!) updates to know if this was kept. 
 I believe the idea was that we'd replace the Fisher Tower data with HARV data once it was available -- this has not happened (but could, however, I don't think it is worth it for the DC lessons).

Reply:
Thanks for that info @mjones01 and @lwasser. I'm not sure what that means in terms of changes to the language of the data description. Could you recommend some specific language changes so I get the attributions right?